t/data/hotplugging-junit-results-softfail.xml

```diff
@@ -0,0 +1,65 @@
+<testsuites id="0" error="n/a" failures="0" softfailures="12" name="hotplugging" skipped="0" tests="12" time="02:03:56">
```

Comment:
Can you please reduce this file to only relevant samples for testing? And better use names similar to the existing test fixtures, not something about hotplugging which isn't relevant here

Reply:
Currently there are 12 tests in total in this file. So how many do think is proper ? Then I will reduce to that number. And what is the best name that you suggest ? I will change them all together.

Reply:
* Use as few entries as necessary, e.g. a single one if this is enough
* For the filename how about to just delete "hotplugging" and use `junit-results-softfail.xml`? Similar for the entries within, e.g. just remove "hotplugging"

Reply:
Done. Also did again make test and make coverage. All work and coverage percentage remains.